src/util/symbols.js

```diff
@@ -9,6 +9,13 @@ export const created = '____skate_created';
 // be passed to vdom functions as tag names.
 export const name = '____skate_name';
 
+// Used on the Constructor
```

Comment:
I wonder if these would be better put into where they are used. I've been getting confused lately because I'm not sure where in the codebase a symbol is used, or usage originated. No need to change right now as this is following this convention, but worth thinking about.